### PR TITLE
MFLT-10167 Add docs for Device Summary Schedules

### PR DESCRIPTION
**Pull request by nihalgonsalves**

### CODE REVIEW

**Review by ejohnso49 — APPROVED** [review]
Love it. I liked the added description of the example values in the screenshot. On nit on the image might be to crop out the browser bar at the top?

File: docs/platform/alerts.mdx [hunk 1]
```diff
@@ -55,11 +60,6 @@ clicking on the Alert entry in the Alerts table.
   />
 </p>
 
-On top of the entries in this view, Memfault will send an email every **5
-minutes** summarizing the number of devices that meet this threshold. Given this
-fact, it is best to set up the Alerts so that they are hit rarely to prevent
-abuse of the email inbox.
-
```
Comment (martijnthe) [inline docs/platform/alerts.mdx:62]: Below it still says

> An email will also get sent once every 12 hours.

**Review by martijnthe — COMMENTED** [review]
Note that when a PR is merged, the docs are deployed right away.
No big deal, but generally I would merge it after shipping the feature.

File: docs/platform/alerts.mdx (hunk 1 above)
Comment (nihalgonsalves) [reply docs/platform/alerts.mdx:62]: => #496

**Pull request merged**